arm-sql/2014-04-01/swagger/importExport.json

```diff
@@ -0,0 +1,495 @@
+        },
+        "ImportExportOperationStatusResponse": {
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "$ref": "#/definitions/ImportExportOperationStatusResponseProperties",
+                    "description": "Gets or sets the properties of the response."
+                },
+                "id": {
+                    "type": "string",
+                    "description": "Gets or sets the id returned from the server."
+                },
+                "name": {
+                    "type": "string",
+                    "description": "Gets or sets the name returned from the server."
+                },
+                "type": {
+                    "type": "string",
+                    "description": "Gets or sets the type returned from the server."
+                }
+            },
+            "description": "Response for Azure Sql Import/Export Status operation."
+        },
+        "ImportExportOperationStatusResponseProperties": {
+            "properties": {
+                "requestType": {
+                    "type": "string",
+                    "description": "Gets or sets the request type of the operation."
+                },
+                "requestId": {
+                    "type": "string",
+                    "format": "uuid",
+                    "description": "Gets or sets the request type of the operation."
+                },
+                "serverName": {
+                    "type": "string",
+                    "description": "Gets or sets the name of the Azure SQL Server."
+                },
+                "databaseName": {
+                    "type": "string",
+                    "description": "Gets or sets the name of the Azure SQL Database."
+                },
+                "status": {
+                    "type": "string",
+                    "description": "Gets or sets the status message returned from the server."
+                },
+                "lastModifiedTime": {
+                    "type": "string",
+                    "description": "Gets or sets the operation status last modified time."
+                },
+                "queuedTime": {
+                    "type": "string",
+                    "description": "Gets or sets the operation queued time."
+                },
+                "blobUri": {
+                    "type": "string",
+                    "description": "Gets or sets the blob uri."
+                },
+                "errorMessage": {
+                    "type": "string",
+                    "description": "Gets or sets the error message returned from the server."
+                }
+            },
+            "description": "Response for Azure Sql Import/Export Status operation."
+        },
+        "ImportRequestParameters": {
+            "properties": {
+                "databaseName": {
+                    "type": "string",
```

Comment:
What happens if the databaseName is already used when the user is providing body parameter for this operation Servers_ImportDatabase? Will the server throw an error? 

As per the description of the operation Servers_ImportDatabase
> "Imports a bacpac into a new Azure SQL database. "

Shouldn't the server auto create the database name?